pages/k8s/cdk-addons.md

```diff
@@ -36,8 +36,29 @@ Sourced from: <https://github.com/coredns/deployment.git>
 CoreDNS has been the default DNS provider for Charmed Kubernetes clusters
 since 1.14.
 
-It is possible to use `kube-dns` as the DNS provider, or turn off DNS
-altogether by adjusting the [kubernetes-master configuration][].
+For additional control over CoreDNS, you can also deploy it into the cluster
+using the [CoreDNS Kubernetes operator charm][coredns-charm]. To do so, set
+the `dns-provider` [kubernetes-master configuration][] option to `none` and
```

Comment:
are these supposed to link to /kubernetes/docs/charm-kubernetes-master#configuration? you didn't make a link at the bottom